docs/contributions.rst

```diff
@@ -82,6 +82,8 @@ Defining kinds is the most important task in writing a new parser.
 Once a kind is introduced, we cannot change it because it breaks
 tags file compatibility.
 
+.. TODO: To where shall we refer?
```

Comment:
I wrote the concepts of kinds and roles in ctags(1).

Reply:
I added the cross reference.